### PR TITLE
ignition-plugin2: new formula

### PR DESCRIPTION
**Pull request by scpeters**
Add dependency on ignition-utils1. Needed by https://github.com/ignitionrobotics/ign-plugin/pull/55

### CODE REVIEW

**Review by adlarkin — COMMENTED** [review]
LGTM. I'll let someone else take a look at this as well before merging since I don't have too much experience with homebrew formulas

**Review by chapulina — APPROVED** [review]

**Pull request merged**
